pkg/apiclients/testapi/2024-10-15/spec.yaml

```diff
@@ -801,33 +782,32 @@ components:
         Test subject representing a source tree located in a Git repository that
         has a Snyk SCM integration.
       properties:
-        asset_id:
-          description: Asset associated with the tested content.
+        commit_id:
```

Comment:
Question: Is the commit id really a uuid? seems like a mistake to me, shouldn't it be just a string?